docs/key.md

```diff
@@ -18,7 +18,7 @@ There are 3 permission settings for a key:
 * both (read and write)
 
 To create a key with a desired permission, run:
-`google-cloud-gemserver create-key --permissions [read/write/both] --use-proj PROJECT_ID`, i.e. `google-cloud-gemserver create-key permissions write --use-proj PROJECT_ID`. By default, if the permissions argument is not given a key will be generated with both (read, write) permissions.
+`google-cloud-gemserver create-key --permissions [read|write|both] --use-proj PROJECT_ID`. By default, if the permissions argument is not given a key will be generated with both (read, write) permissions.
```

Comment:
Actually, you should probably put the square brackets around --permissions also, e.g. `[--permissions read|write|both]` (and put it at the end of the arg list, after --use-proj) because that is the scope of what is optional in the command line.